### PR TITLE
Fixes Broken Pages

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes Broken Pages due to #1094 

#### Describe the changes you have made in this PR -
I have changed some alt= to alt: 

Please let me know if anymore changes are required here.

### CODE REVIEW

**Comment by Shivansh2407** [comment]
Fixed Now @tachyons .
Capture

File: app/views/logix/examples.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/3/projects/247" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/fullAdder.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/fullAdder.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/fullAdder.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 2]
```diff
@@ -33,7 +33,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/3/projects/248" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/RippleCarry.jpeg", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/RippleCarry.jpeg", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/RippleCarry.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 3]
```diff
@@ -44,7 +44,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/3/projects/249" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/AsyncCounter.jpeg", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/AsyncCounter.jpeg", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/AsyncCounter.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 4]
```diff
@@ -55,7 +55,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank"  href="https://circuitverse.org/users/3/projects/250" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/Keyboard.jpeg", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/Keyboard.jpeg", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/Keyboard.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 5]
```diff
@@ -66,7 +66,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank"  href="https://circuitverse.org/users/3/projects/251" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/FlipFlop.jpeg", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/FlipFlop.jpeg", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/FlipFlop.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 6]
```diff
@@ -78,7 +78,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/126/projects/252" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/ALU.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/ALU.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/ALU.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/projects/get_projects.html.erb [hunk 7]
```diff
@@ -31,7 +31,7 @@
         </div>
       </div>
       <div class="col-md-3">
-         <%= image_tag "<%= project.image_preview.url %>", :class=>"img-fluid img-thumbnail", alt="<%= project.name %>" %>
+         <%= image_tag  project.image_preview.url , :class => "img-fluid img-thumbnail", alt: project.name %>
```
Comment (tachyons) [inline app/views/projects/get_projects.html.erb]:
```suggestion
         <%= image_tag  project.image_preview.url , class: "img-fluid img-thumbnail", alt: project.name %>
```

File: app/views/logix/features.html.erb [hunk 8]
```diff
@@ -47,7 +47,7 @@
                      <p class="card-text">CircuitVerse can export high resolution images in multiple formats including SVG.</p>
                  </div>
                  <div class="col-12 col-md-8 col-lg-8">
-                  <%= image_tag "img/Control Sequencer.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/Control Sequencer.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/features.html.erb]:
```suggestion
                  <%= image_tag "/img/Control Sequencer.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/features.html.erb [hunk 9]
```diff
@@ -34,10 +34,10 @@
                      <p class="card-text">Create subcircuits once and use them repeatedly. This allows easier and more structured design. Future versions will allow importing of subcircuits from external sources. To use a subcircuit, simply create and design the subcircuit as a normal circuit and Label inputs and outputs. Then after that create another circuit within the same project and click circuit -> insert subcircuit and select the subcircuit.</p>
                  </div>
                  <div class="col-12 col-md-2">
-                  <%= image_tag "img/halfAdder.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/halfAdder.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
                  </div>
                  <div class="col-12 col-md-4">
-                  <%= image_tag "img/fullAdder.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/fullAdder.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/features.html.erb]:
```suggestion
                  <%= image_tag "/img/fullAdder.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/features.html.erb [hunk 10]
```diff
@@ -34,10 +34,10 @@
                      <p class="card-text">Create subcircuits once and use them repeatedly. This allows easier and more structured design. Future versions will allow importing of subcircuits from external sources. To use a subcircuit, simply create and design the subcircuit as a normal circuit and Label inputs and outputs. Then after that create another circuit within the same project and click circuit -> insert subcircuit and select the subcircuit.</p>
                  </div>
                  <div class="col-12 col-md-2">
-                  <%= image_tag "img/halfAdder.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/halfAdder.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/features.html.erb]:
```suggestion
                  <%= image_tag "/img/halfAdder.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/features.html.erb [hunk 11]
```diff
@@ -24,7 +24,7 @@
                           faster and uncluttered. To change the bitWidth of the circuit elements, Just select the circuit element and change the bitwidth value in the property menu.</p>
                  </div>
                  <div class="col ">
-                  <%= image_tag "img/bus.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/bus.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/features.html.erb]:
```suggestion
                  <%= image_tag "/img/bus.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/features.html.erb [hunk 12]
```diff
@@ -9,7 +9,7 @@
                     <!-- <p><a class="btn btn-dark btn-lg" href="#" role="button">Launch Simulator &raquo;</a></p> -->
                 </div>
                 <div class="col">
-                  <%= image_tag "img/logixBanner.png", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                  <%= image_tag "/img/logixBanner.png", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/features.html.erb]:
```suggestion
                  <%= image_tag "/img/logixBanner.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

File: app/views/logix/examples.html.erb [hunk 13]
```diff
@@ -90,7 +90,7 @@
                           <a class="btn btn-dark btn-lg" target="_blank"  href="https://circuitverse.org/users/3/projects/67" role="button">View Example &raquo;</a>
                  </div>
                  <div class="col">
-                     <%= image_tag "img/SAP.jpeg", :class=>"img-fluid img-thumbnail", alt="Responsive image" %>
+                     <%= image_tag "/img/SAP.jpeg", :class => "img-fluid img-thumbnail", alt:"Responsive image" %>
```
Comment (tachyons) [inline app/views/logix/examples.html.erb]:
```suggestion
                     <%= image_tag "/img/SAP.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
```

**Pull request merged**